contracts/LATokenMinter.sol

```diff
@@ -135,14 +137,14 @@ contract LATokenMinter is SafeMath {
         helper = _helperAddress;
         token = LATToken(_LATTokenAddress);
 
-        numberOfDays = 5 * 365; // 5 years
+        numberOfDays = mul(5, 365); // 5 years
```

Comment:
just out of curiosity, what about leap days? not all years have 365 days

Reply:
This will only be 1-2 days so probably not too much of an issue, but good to be aware of